WalletWasabi.Fluent/ViewModels/Wallets/Send/SendViewModel.cs

```diff
@@ -192,7 +192,7 @@ private async Task OnPasteAsync(bool pasteIfInvalid = true)
 			Uri.IsWellFormedUriString(endPoint, UriKind.Absolute))
 		{
 			var payjoinEndPointUri = new Uri(endPoint);
-			if (!Services.PersistentConfig.UseTor)
+			if (!Services.HttpClientFactory.IsTorEnabled)
```

Comment:
@lontivero is this OK?

Reply:
The same issue can occur elsewhere in the code?

Reply:
No, it's not okay because `Config` is the effective config. `PersistentConfig` is simply what is in the config file.

Reply:
As you say @turbolay, `PersistentConfig` is wrong, that's clear. `Config` is the effective configuration, the real one, the one that matters. 

This means that you have to use `Config` instead of `PersistentConfig`. What you did instead is to use `HttpClientFactory.IsTorEnabled` which if you track it a loong way back you will see that after some convoluted logic it takes its value from `Config.UseTor`  . So, my point is that even when in the end result is the same, the correct thing is to take the config values from the Config instance.